Lua/Libraries/CYK/EntityManager.lua

```diff
@@ -1,5 +1,8 @@
 -- This library handles everything related to entities
 -- No wonder why it got its current name!
+
+
```

Comment:
Why those extra lines?

Reply:
I think it's for when I tried to use the `getset` library in this file. I put the line here, deleted it later but forgot to remove the two lines around it

Reply:
Yeet 'em then, if they're relics of an old attempt, please.

Reply:
I'm on it